client/src/app/pages/vulnerability-list/vulnerability-table.tsx

```diff
@@ -74,11 +76,7 @@ export const VulnerabilityTable: React.FC = () => {
                         {item.identifier}
                       </NavLink>
                     </Td>
-                    <Td
-                      width={45}
-                      modifier="truncate"
-                      {...getTdProps({ columnKey: "title" })}
-                    >
+                    <Td width={45} {...getTdProps({ columnKey: "title" })}>
```

Comment:
The `modifier=truncate` has been removed. Any particular reason? Initially "truncate" was set expecting to have long strings there and render a tooltip.

Ahh it might be that V1 didn't have that column truncated? But as far as I remember, in v1 we didn't apply truncate not because we didn't want to but because we couldn't do it so it was a limitation and not what we desired to have.

Reply:
@carlosthe19916 yep the reasoning was because it wasn't there in v1, but also sometimes large chunks of text having a big tooltip felt awkward, but I will put it back because I agree with you, and think is does provide more benefit than any inconvenience